### PR TITLE
Feature implement oversight page fix

### PR DESCRIPTION
**Pull request by fldata22**

### CODE REVIEW

**Comment by ahadziii** [comment]
Says there are conflicts

**Comment by jaedag** [comment]
@fldata22 conflicts?

**Comment by fldata22** [comment]
> @fldata22 conflicts?

all sorted now

File: api/src/resolvers/aggregates/component-service-cypher.ts [hunk 1]
```diff
@@ -76,3 +76,15 @@ export const componentGatheringServiceServiceAggregates = `
 
 RETURN week AS week, SUM(componentRecords.attendance) AS attendance, SUM(componentRecords.income) AS income ORDER BY week DESC LIMIT toInteger($limit)
 `
+export const componentOversightServiceAggregates = `
+ MATCH (church:Oversight {id:$id})
+  
+  MATCH (church)-[:HAS_HISTORY]->(:ServiceLog)-[:HAS_COMPONENT*1..5]->(componentServices:ServiceLog)
```
Comment (jaedag) [inline api/src/resolvers/aggregates/component-service-cypher.ts:82]: Why HAS_COMPONENT 1..5?

Comment (jaedag) [reply api/src/resolvers/aggregates/component-service-cypher.ts:82]: ??

File: web-react-ts/src/pages/directory/quick-facts/this-month/GatheringServiceAvgWeekdayQuickFacts.jsx [hunk 2]
```diff
@@ -48,7 +48,7 @@ const GatheringServiceAvgWeekdayQuickFacts = () => {
       leadersName: `${gatheringServiceAttendance?.leader?.firstName} ${gatheringServiceAttendance?.leader?.lastName}`,
```
Comment (jaedag) [inline web-react-ts/src/pages/directory/quick-facts/this-month/GatheringServiceAvgWeekdayQuickFacts.jsx:48]: Why a jsx file here?

File: web-react-ts/src/pages/directory/reusable-forms/GatheringServiceForm.jsx [hunk 3]
```diff
@@ -0,0 +1,223 @@
+import { useMutation, useQuery } from '@apollo/client'
```
Comment (jaedag) [inline web-react-ts/src/pages/directory/reusable-forms/GatheringServiceForm.jsx:1]: Another jsx file

**Review by jaedag — CHANGES_REQUESTED** [review]

File: web-react-ts/src/pages/directory/grids/OversightMembers.tsx [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+// import TabletDesktopView from 'components/responsive-design/TabletDesktopView'
+
+const OversightMembers = () => {
+  const { oversightId } = useContext(ChurchContext)
+  const { data, loading, error } = useQuery(GET_OVERSIGHT_MEMBERS, {
+    variables: { id: oversightId },
+  })
+
+  return (
+    console.log(data),
```
Comment (jaedag) [inline web-react-ts/src/pages/directory/grids/OversightMembers.tsx:15]: Why a console log?

**Review by jaedag — CHANGES_REQUESTED** [review]

File: web-react-ts/src/pages/directory/grids/OversightMembers.tsx [hunk 5]
```diff
@@ -0,0 +1,24 @@
+import React, { useContext } from 'react'
+import { useQuery } from '@apollo/client'
+import MembersGrid from '../../../components/members-grids/MembersGrid'
+import { GET_OVERSIGHT_MEMBERS } from './GridQueries'
+import { ChurchContext } from '../../../contexts/ChurchContext'
+// import TabletDesktopView from 'components/responsive-design/TabletDesktopView'
```
Comment (jaedag) [inline web-react-ts/src/pages/directory/grids/OversightMembers.tsx:6]: What's this comment about?

**Review by jaedag — CHANGES_REQUESTED** [review]

**Review by jaedag — APPROVED** [review]
This last commit message is not a good one. I'm letting it slide because you have worn me out with this PR

**Pull request merged**